src/pkg/packaging/rpm/scripts/after_install_host.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
My gut tells me for these preinstall and post install scripts we should instead be using `#!/bin/sh` so we don't have a dependency on bash.

@omajid is that shebang reasonable or should we be using `#!/usr/bin/env sh`